.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -5,11 +5,9 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking the time to file a bug report!
+        Thank you for taking the time to report a bug!
 
-        Before posting, please check that the bug hasn't already been:
-        * [fixed in the next release](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
-        * [discussed previously](https://github.com/search/advanced?r=stylelint%2Fstylelint&type=Issues)
+        Please [search our issues](https://github.com/search/advanced?r=stylelint%2Fstylelint&type=Issues) to check that the bug hasn't already been reported.
```

[suggest] When the advance search, the `type` parameter seems unnecessary:

```suggestion
        Please [search our issues](https://github.com/search/advanced?r=stylelint%2Fstylelint) to check that the bug hasn't already been reported.
```

Or, if the normal search, using `type` makes sense:

```suggestion
        Please [search our issues](https://github.com/search?q=repo%3Astylelint%2Fstylelint&type=issues) to check that the bug hasn't already been reported.
```